Lib/test/test_capi/test_opt.py

```diff
@@ -678,7 +678,7 @@ def testfunc(n):
         self.assertLessEqual(len(guard_nos_float_count), 1)
         # TODO gh-115506: this assertion may change after propagating constants.
         # We'll also need to verify that propagation actually occurs.
-        self.assertIn("_BINARY_OP_ADD_FLOAT", uops)
```

Comment:
Do we also want to add `self.assertNotIn("_BINARY_OP_ADD_FLOAT", uops)` or is it not needed?